jobs/catalog.go

```diff
@@ -0,0 +1,29 @@
+package jobs
+
+import (
+	"github.com/flanksource/duty/job"
+)
+
+var RefreshCatalogAnaylsisChangeCount7dView = &job.Job{
+	Name:       "RefreshCatalogAnaylsisChangeCount7dView",
+	Schedule:   "@every 10m",
+	Retention:  job.RetentionFew,
+	Singleton:  true,
+	JobHistory: true,
+	RunNow:     true,
+	Fn: func(ctx job.JobRuntime) error {
+		return job.RefreshConfigItemAnalysisChangeCount7d(ctx.Context)
+	},
+}
+
+var RefreshCatalogAnaylsisChangeCount30dView = &job.Job{
```

Comment:
@moshloop When some first sets up the instance, the materialized view will be empty, and say they run the scrapers etc. after 15 mins, they would have to wait for an hour for catalog page to work since in that case it would be empty